modules/internal/ChapelBase.chpl

```diff
@@ -2234,8 +2234,16 @@ module ChapelBase {
   inline operator :(x:chpl_anyreal, type t:chpl_anyreal) do
     return __primitive("cast", t, x);
 
+  proc chpl_checkCastAbstractEnumError(type enumType, type dstType) param do
+    if isAbstractEnumType(enumType) then
+      compilerError("can't cast from an abstract enum ('" +
```

Comment:
How about we drop the `()` and contraction: "cannot cast abstract enum type 'A' to 'B'" instead?

Reply:
I modeled this error message after the existing error message for casting abstract enums,
`compiler/resolution/functionResolution.cpp:4756`.

I have no problem making this change, but I would prefer to be consistent and change both.

Reply:
The parens feel out of place with what we do for other error messages, but if changing both is too much of a PITA then you can keep as is.

Reply:
I would rather just do the "right" thing and fix all the error messages, which is what I am doing right now. I will rerun full paratests to make sure I did not inadvertently break things by rewording the error messages